schemas/event.yml

```diff
@@ -14,7 +14,7 @@
     A metric is defined as an event containing one or more numerical or
     categorical measurements and the time at which the measurement was taken.
     Examples of metric events include memory pressure measured on a host,
-    or vulnerabilities measured on a scanned host.
+    or vulnerabilities measured on a scanned host. See `event.kind` for additional details.
```

Comment:
We still have wording of putting vulnerability scans in `event.kind:metric` here

Reply:
fixed, thanks.